Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11
+
+WORKDIR  /
+
+COPY requirements.txt ./
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+EXPOSE 8000
+
+CMD ["uvicorn", "src.blog.main:app", "--reload"]
```

Comment:
I add 'src' because I couldn't run docker compose file It gives me some kind of error and I changed the whole directory names all import file system the whole project. I thought we made a mistake and forget to add 'src' and this is reason why we get an error. But when I saw your branch It can be done without 'src' as well. I will change all imports again.